content/nosotros/contents+en.lr

```diff
@@ -0,0 +1,207 @@
+body:
+
+#### heading ####
+title: About us
+#### section-block ####
+title: 
+----
+body:
+
+##### two-columns #####
+class_left: col-lg-6 col-md-6 col-sm-6
+-----
+class_right: col-lg-6 col-md-6 col-sm-6
+-----
+left:
+
+###### image ######
+url: /img/meetup-data-science-new-age.jpg
+------
+alt: PyBAQ Team, PyCon 2018
+------
+class: img-responsive
+-----
+right:
+
+###### text-block ######
+text:
+
+#### More about us
+
+We are a community of web developers, devops, endeavors and students.
+
+Most of us have our own companies while the others work in the companies of these colleagues.
+Another group of djangonauts is still in college or they follow their own path, making their way into the world of entrepreneurship or freelance. 
```

Comment:
Another group is still in college or they follow their own path, making their way into the world of entrepreneurship or freelance.